public/js/NeighbourImages.js

```diff
@@ -0,0 +1,55 @@
+function NeighbourImages(query_id) {
+    this.neighbours = [];
+    this.selectedImage = "";
+    this.neighbourImages = [];
+}
+
+NeighbourImages.prototype.getImages = function(query_id) {
+    var self = this;
+
+    $.when(
+        $.ajax({
+            url: '/api/sample/neighbours/' + query_id,
+            async: false,
+            success: function(json) {
+                self.neighbours = json[0].neighbours.slice(1, 25);
```

Comment:
The number of neighbours is another magic number... Is it referenced elsewhere, and if so, a global config file for it would be nice...